doc/whats_new/v1.1.rst

```diff
@@ -60,13 +60,23 @@ Changelog
   add this information to the plot.
   :pr:`21038` by :user:`Guillaume Lemaitre <glemaitre>`.
 
+:mod:`sklearn.cluster`
+..........................
+
 - |Enhancement| :class:`cluster.SpectralClustering` and :func:`cluster.spectral`
   now include the new `'cluster_qr'` method from :func:`cluster.cluster_qr`
   that clusters samples in the embedding space as an alternative to the existing
   `'kmeans'` and `'discrete'` methods.
   See :func:`cluster.spectral_clustering` for more details.
   :pr:`21148` by :user:`Andrew Knyazev <lobpcg>`
 
+- |Efficiency| In :class:`cluster.KMeans`, the default ``algorithm`` is now
+  ``"lloyd"`` which is the full classical EM-style algorithm. Both ``"auto"``
+  and ``"full"`` are deprecated, and they are aliases for ``"lloyd"``. The
+  previous default was ``"auto"`` which used Elkan's algorithm, but it is
+  often slower, and its results are identical, hence the change.
+  :pr:`21735` by :user:`Aurélien Geron <ageron>`.
```

Comment:
Since we change the default behavior I would also mention that by default the Lloyd algorithm is used instead of Elkan for `cluster.KMeans` in the  "Changed models" section near the top of the changelog. Have a look at previous versions changelogs to see what kind of info we typically put in this section.

Reply:
Done